kubetest/e2e.go

```diff
@@ -623,10 +623,35 @@ func kubemarkTest(testArgs []string, dump, numNodes string) error {
 		})
 	}
 
+	if err := control.XmlWrap(&suite, "kubectl version", getKubectlVersion); err != nil {
+		return err
+	}
+
+	if err := control.XmlWrap(&suite, "IsUp", deploy.IsUp); err != nil {
```

Comment:
Also what's the purpose of this? This doesn't seem to check sth new IIUC.

Reply:
I'm doing 1-to-1 port, this was a step in old behavior which is checked by the bash deployer